configuration/yaml.md

```diff
@@ -0,0 +1,660 @@
+| `tags`            | The list of tags. This can contain both semantic tags and non-semantic tags.                                                                                                                                                                                                     |
+| `format`          | A short form for the `stateDescription` `pattern` metadata. This is the equivalent of the format enclosed in square brackets in DSL's item label.                                                                                                                                |
+| `unit`            | A short form for the `unit` metadata, to define the item's unit.                                                                                                                                                                                                                 |
+| `autoupdate`      | A short form for the `autoupdate` metadata. Valid values are: `true` or `false`.                                                                                                                                                                                                 |
+| `channel`         | A short form for the `channels` list to define a linked channel UID for when there is only one channel with a blank config.                                                                                                                                                      |
+| `channels`        | A key-value map to define the channels linked from this item. The key is the channel uid to link to, and the value is the link configuration as a key-value map. When no configuration is required, an empty map `{}` must be used as the value.                                 |
+| `metadata`        | A key-value map to define item metadata. The metadata defined in here overrides any short forms when both are specified.                                                                                                                                                         |
+
+Example:
+
+```yaml
+version: 1
+
+items:
+  BedRoom1_Switch:
+    type: Group
+    label: Bedroom1 Light Bulb
+    groups:
+      - lBedroom1
+    tags:
+      - Lightbulb
+
+  BedRoom1_Light_Switch:
+    type: Switch
+    label: Bed Room1 Light Switch
+    icon: switch
+    format: '%s'
+    groups:
+      - BedRoom1_Light_Bulb
+    tags:
+      - Switch
+    channel: mqtt:topic:bedroom1-switch:switch1
+
+  BedRoom1_Switch_StatusLED:
+    type: Switch
+    groups:
+      - BedRoom1_Switch_Group
+    channels:
+      mqtt:topic:bedroom1-switch:statusled:
+        # The configuration of this item-channel link is defined below
+        profile: transform:MAP
+        function: "|open=ON;closed=OFF" # | is a YAML special character, so enclose it in quotes.
```

Comment:
what does "|" do?

Reply:
Again, this will be a part of a more detailed explanation about YAML syntax. Officially it's https://yaml.org/spec/1.1/.